pyiron_atomistics/vasp/base.py

```diff
@@ -2075,9 +2075,9 @@ def collect(self, directory=os.getcwd(), sorted_indices=None):
         if vasprun_working:
             log_dict["forces"] = self.vp_new.vasprun_dict["forces"]
             log_dict["cells"] = self.vp_new.vasprun_dict["cells"]
-            log_dict["volume"] = np.array([
-                np.linalg.det(cell) for cell in self.vp_new.vasprun_dict["cells"]
-            ])
+            log_dict["volume"] = np.array(
+                [np.linalg.det(cell) for cell in self.vp_new.vasprun_dict["cells"]]
+            )
```

Comment:
Can this be simplified to: 
```suggestion
            log_dict["volume"] = np.linalg.det(self.vp_new.vasprun_dict["cells"])
```

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.linalg.det.html